Resources/Locale/en-US/headset/headset-component.ftl

```diff
@@ -1,11 +1,22 @@
 # Chat window radio wrap (prefix and postfix)
 chat-radio-message-wrap = [color={$color}]{$channel} {$name} says: "{$message}"[/color]
 
+examine-encryption-key = It is providing these frequencies to the headset:
+examine-encryption-key-channel = [color={$color}]:{$key} for {$id} ({$freq})[/color]
```

Comment:
```suggestion
examine-encryption-key-channel = [color={$color}]{$key} for {$id} ({$freq})[/color]
```

Reply:
For ; of common already solved, so no need for it